plugins/woocommerce-admin/client/marketplace/components/product-card/product-card-footer.tsx

```diff
@@ -1,46 +1,92 @@
 /**
  * External dependencies
  */
-import { Icon } from '@wordpress/components';
+import { Button, Icon } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
+import { useEffect, useState } from '@wordpress/element';
+import { recordEvent } from '@woocommerce/tracks';
+import { navigateTo, getNewPath } from '@woocommerce/navigation';
+
+/**
+ * Internal dependencies
+ */
+import { Product } from '~/marketplace/components/product-list/types';
+import { getAdminSetting } from '~/utils/admin-settings';
 
 export interface ProductCardFooterProps {
-	price: number | null | undefined;
-	currencySymbol: string;
-	averageRating: number | null | undefined;
-	reviewsCount: number | null | undefined;
+	product: Product;
 }
 
 function ProductCardFooter( props: ProductCardFooterProps ) {
-	const { price, currencySymbol, averageRating, reviewsCount } = props;
+	const { product } = props;
+	const [ isInstalled, setIsInstalled ] = useState( false );
+
+	useEffect( () => {
+		const wccomSettings = getAdminSetting( 'wccomHelper', {} );
+		const installedProducts: string[] = wccomSettings?.installedProducts;
+
+		const result = !! installedProducts.find(
+			( item ) => item === product.slug
+		);
+
+		setIsInstalled( result );
+	}, [ product ] );
+
+	function openInstallModal() {
+		recordEvent( 'marketplace_add_to_store_clicked', {
+			product_id: product.id,
+		} );
+
+		navigateTo( {
+			url: getNewPath( {
+				installProduct: product.id,
+			} ),
+		} );
+	}
+
+	// We hardcode this for now while we only display prices in USD.
+	const currencySymbol = '$';
+
+	if ( product.isInstallable && ! isInstalled ) {
```

Comment:
I think we need one more check here: seeing this button doesn't guarantee that a user can install it. Installing a plugin depends on the `install_plugins` permission. While we check `manage_woocommerce` to load the page and authenticate APIs, there might a chance that the `install_plugins` cap may not be handed out to the store manager.

Reply:
Oh nice. I'll try this tomorrow and report back